pontoon/base/views.py

```diff
@@ -341,24 +327,68 @@ def contributor(request, email, template='user.html'):
     return render(request, template, data)
 
 
-def contributors(request, template='users.html'):
-    """Top contributors view."""
-    log.debug("Top contributors view.")
+class ContributorsMixin(object):
+    def contributors_filter(self, **kwargs):
+        """
+        Method returrns filters for given contributors page. Method is dedicated to use 
```

Comment:
Mispelled "returns". No need to mention about overriding, either. "Return Q() filters for fetching contributors. Fetches all by default." would be enough.